models/test/unittests.yml

```diff
@@ -464,6 +474,8 @@ RecordContainingGenericRecords<A, B>: !record
     g2a: RecordWithAliasedOptionalGenericUnionField<A, B>
     g3: MyTuple<A, B>
     g3a: AliasedTuple<A, B>
+    g4: RecordWithGenericVectors<B>
```

Comment:
We should also test maps and dynamic arrays

Reply:
NP, I already did that in my local example. Adding now.